modules/rtm/tests/quicktest.R

```diff
@@ -0,0 +1,20 @@
+## Quick test
```

Comment:
OK. Like I said, I have no clue how testthat works, but I'll look into it. I wrote these scripts for testing the inversion quickly (just hitting "Source" rather than subsetting the data and organizing the results manually). I'll keep these here in their present form for now because they are very useful for debugging, but eventually, I'll restructure them as vignettes and move elsewhere.